package.json

```diff
@@ -41,7 +41,7 @@
     "pelias-config": "2.12.0",
     "pelias-dbclient": "2.2.2",
     "pelias-logger": "0.2.0",
-    "pelias-model": "5.0.1",
+    "pelias-model": "pelias/model",
```

Comment:
whats this?

Reply:
Oh you're probably waiting for this to be merged, right? https://github.com/pelias/model/pull/75

Reply:
Correct, I'm stalling on merging that for a bit as I worked out how this effects the whole ecosystem.